actors/builtin/miner/partition_state.go

```diff
@@ -537,25 +537,25 @@ func NewPowerPair(raw, qa abi.StoragePower) PowerPair {
 	return PowerPair{Raw: raw, QA: qa}
 }
 
-func (pp *PowerPair) IsZero() bool {
```

Comment:
Any reason to change this?

Reply:
Yes, when PowerPair is returned from a function Go refuses to call pointe-receiver methods on it.

`cannot call pointer method on actor.powerPairForSectors(bothSectors)`